src/scss/components/file/mixins/hxFileControl.scss

```diff
@@ -0,0 +1,9 @@
+@import "../config";
```

Comment:
This `@import` _should_ be able to removed.

Reply:
Changes have been done, please review.